collect_app/src/main/java/org/odk/collect/android/formentry/ODKView.java

```diff
@@ -308,6 +309,14 @@ private View getDividerView() {
         divider.setBackgroundResource(new ThemeUtils(getContext()).getDivider());
         divider.setMinimumHeight(3);
 
+        ViewGroup.MarginLayoutParams params = new ViewGroup.MarginLayoutParams(
+                ViewGroup.LayoutParams.WRAP_CONTENT,
+                ViewGroup.LayoutParams.MATCH_PARENT
+        );
+        int marginVertical = (int) getContext().getResources().getDimension(org.odk.collect.androidshared.R.dimen.margin_small);
```

Comment:
I did not realise that `getDimension` would give us back px rather than dp here. I guess that's because the dimension is defined as `8dp` rather than just `8`.